updating CDL integration to add ThreatName to two threat log queries (#25754)
demisto/content: Packs/CortexDataLake/ReleaseNotes/1_4_4.md

#### Integrations
##### Cortex Data Lake XSOAR Connector

> - Updated the output fields for the ***cdl-query-threat-logs*** and the ***cdl-get-critical-threat-logs*** commands to include Threat Name.

Reviewer: ```suggestion
- Updated the outputs of the ***cdl-query-threat-logs*** and the ***cdl-get-critical-threat-logs*** commands to include Threat Name.
```

> ##### Cortex Data Lake XSOAR Connector
> - Updated the output fields for cdl-query-threat-logs and cdl-get-critical-threat-logs query to include Threat Name.
> - Updated the integration to include the Threat Name value in the function outputs for cdl-get-critical-threat-logs and cdl-query-threat-logs.

Reviewer: ```suggestion
##### Cortex Data Lake XSOAR Connector
- Updated the output fields for the ***cdl-query-threat-logs*** and the ***cdl-get-critical-threat-logs*** commands to include Threat Name.
- Updated the docker image to: *demisto/python_pancloud_v2:1.0.0.53496*.
```
